### ISSUE TITLE
Event Single Page - newly added friday event cannot be seen in the sidebar upcoming events list. 

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Evolution-MMA#170]
I have scrolled all the upcoming events list and i cannot find the Friday event .

![image](https://cloud.githubusercontent.com/assets/12791909/10215700/6c127930-6856-11e5-839b-63cf36f4ce10.png)

**Comment by christiancebudigital** [comment]
Hi Erma,

You need to add category in each event you will be creating in order the event to show up in the sidebar,
See this link: http://www.screenr.com/nBBN

Thank,
Christian

**Comment by ermatitan** [comment]
Hello Christian, 

I have tried to add categories in each event, but I just notice that:

Friday Madness Event : there are 3 schedule events : Friday 8AM-9AM , Friday 2PM, Sat- 2PM and the Sat 2PM is not listed on the sidebar list of events. I have checked also the backend of this event, and no category was listed but the two events was already there. Kinda it confused me.

I have added again a new Event with a category: 

Saturday Zumba  with event category of Physical Fitness and was schedule at Sat: 9Am-10PM and 17:00 - 18:00. And only the event "17:00-18:00" were listed.
